fastlane/actions/get_latest_github_release.rb

```diff
@@ -0,0 +1,170 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      GET_LATEST_GITHUB_RELEASE_CUSTOM_VALUE = :GET_LATEST_GITHUB_RELEASE_CUSTOM_VALUE
+    end
+
+    class GetLatestGithubReleaseAction < Action
+      def self.run(params)
+        UI.message("Getting latest release on GitHub (#{params[:server_url]}/#{params[:url]})")
+
+        GithubApiAction.run(
+          server_url: params[:server_url],
```

Comment:
Hmm unless we pretend to customize this for any reason, I would just hardcode things that will not change like the `server-url`. Also, we probably will use an `api_token` only, so I think we can remove the `api_bearer` parameter

Reply:
I based this action from `get_github_release` https://docs.fastlane.tools/actions/get_github_release/ and thought we could maybe publish it initially, but I think it's better to simplify it and just do as you say and remove all this stuff we are not using